client/scss/components/_panel.scss

```diff
@@ -81,6 +97,41 @@ $header-button-size: theme('spacing.8');
   }
 }
 
+.w-panel__divider {
+  flex: 1;
+}
+
+.w-panel__controls {
+  margin: calc(-1 * theme('spacing.4'));
+  margin-inline-end: calc(-1 * theme('spacing.8'));
+  margin-inline-start: 0;
+  padding: theme('spacing.4');
+
+  @include media-breakpoint-up(sm) {
+    margin: calc(-1 * theme('spacing.4'));
+    margin-inline-start: 0;
+  }
+
+  > * {
+    visibility: hidden;
+  }
+
+  > .w-panel__controls-cue {
+    position: absolute;
+    visibility: visible;
+  }
+
+  @media (hover: hover) {
```

Comment:
Sorry if I'm reading this wrong, but it looks like we only add the visible state for children of w-panel__controls on devices that support hover? Do we have an option for devices that don't?

Reply:
Indeed! This should be fixed now, with the above styles moved to be within `@media (hover: hover)` so they only apply for devices with hover support.